src/dcast.d

```diff
@@ -3182,7 +3182,12 @@ Lagain:
     {
         // Bugzilla 13841, all vector types should have no common types between
         // different vectors, even though their sizes are same.
-        goto Lincompatible;
+        auto tv1 = cast(TypeVector)t1;
+        auto tv2 = cast(TypeVector)t2;
```

Comment:
(imho: names that are used only once for simple expressions are seldom worth introducing)